lib/checks/color/color-contrast.json

```diff
@@ -2,9 +2,23 @@
 	"id": "color-contrast",
 	"evaluate": "color-contrast-evaluate",
 	"options": {
-		"noScroll": false,
 		"ignoreUnicode": true,
-		"ignoreLength": false
+		"ignoreLength": false,
+		"boldValue": 700,
+		"boldPt": 14,
+		"nonBoldPt": 18,
+		"contrastRatio": {
+			"normal": {
+				"expected": 4.5,
+				"minThreshold": 0,
+				"maxThreshold": 999
```

Comment:
I think we should allow this to be undefined, and if it is, ignore it. Same for the minThreshold.

```suggestion
```